tested/dsl/translate_parser.py

```diff
@@ -148,6 +168,7 @@ def _parse_yaml(yaml_stream: str) -> YamlObject:
             yaml.add_constructor("!" + actual_type, _custom_type_constructors, loader)
     yaml.add_constructor("!expression", _expression_string, loader)
     yaml.add_constructor("!oracle", _return_oracle, loader)
+    yaml.add_constructor("!path", _path_string, loader)
```

Comment:
I am not against adding another custom tag, but I do think we need to be slightly careful here, since we also expose all "TESTed types" as tags, this would prevent us from ever having a "path" type in TESTed.

An alternative is working with plain objects, e.g.

```yaml
- path: "animal.txt" 
  content:
     type: "path"
     path: "media/workdir/animal.txt" 
```

Again, not against it, just something to consider.

Reply:
has this ever been discussed during one of your thesis meetings?